LongWarOfTheChosen/Src/LW_Overhaul/Classes/XComGameState_LWOutpost.uc

```diff
@@ -245,6 +248,7 @@ function StateObjectReference CreateRebel(XComGameState NewGameState, XComGameSt
 	local array<X2StrategyElementTemplate> PersonalityTemplates;
 	local StateObjectReference NewUnitRef;
 	local float FacelessChance;
+	local name PsionAbilityName;
```

Comment:
Where is that set? What's the Connection between PsionAbilityName and FacelessReductionPsiAbilities?